### DISCUSSION TITLE
Any plans to bring spring data repository interface2code

### DISCUSSION THREAD

**Discussion by daxsieger** [quarkusio/quarkus#24446] [Community]
Hi, i'm new to quarkus and i'd like to see something like the old good spring data facility that turns fairly named methods to jpa queries at runtime. Is there anything in place? I think it may pose some extra challenge to bring it also to native implementations but it might be possible to convert method names to panache queries or simply plain jql. 

Is anybody working on it? if not, would it be useful?

**Comment by maxandersen** [comment]
I believe @FroMage looked at it at some point. Best if he comments on the current state of things.

**Comment by loicmathieu** [comment]
Yes, it exists, it's a compatibility layer on top of Spring Data that translate named method to Panache Queries and use Hibernate ORM with Panache, see https://quarkus.io/guides/spring-data-jpa

For question about it, @geoand is your man ;)
